packages/block-editor/src/hooks/color.js

```diff
@@ -137,6 +139,19 @@ export function addSaveProps( props, blockType, attributes ) {
 	);
 	props.className = newClassName ? newClassName : undefined;
 
+	// Add "has-background-color-light" and "has-background-color-dark" classes,
+	// depending on the luminosity of the selected background color.
+	if ( backgroundColor ) {
+		props.className += tinycolor(
+			getColorObjectByAttributeValues(
+				wp.data.select( 'core/block-editor' ).getSettings().colors,
```

Comment:
This line looks a bit weird to me, but I couldn't find a different way to do it.
Ideally we'd be able to use something like this to get the `colors`:
```js
const colors = useSelect( ( select ) => {
	return select( 'core/block-editor' ).getSettings().colors;
}, [] );
```
but unfortunately that can't be used inside the `addSaveProps` function...
Any ideas?